addons/mail/security/mail_security.xml

```diff
@@ -46,9 +46,7 @@
                 [
                     ('is_self', '=', True),
                     "|",
-                        "&amp;",
-                            ("channel_id.channel_type", "!=", "channel"),
-                            ("channel_id.is_member", "=", True),
+                        ("channel_id.channel_type", "!=", "channel"),
```

Comment:
This is doing a change in noupdate=1 data, so all existing installations won't benefit from this.

Reply:
OK then. Indeed not being a bugfix, it will continue working as it was without this change. On OCA, when we perform these fixes/imp, we prepare a migration script and raise module version, but I know that here you resist more to bump versions and migration scripts.